test/tests/functional/pbs_qmgr.py

```diff
@@ -159,3 +208,143 @@ def test_multi_attributes(self):
+                                runas=ROOT_USER)
+        
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc2", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc3", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1,sc3,default", expect=True,
+                                runas=ROOT_USER)
+            
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="invalide_scname", expect=True,
+                                runas=ROOT_USER)
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="sc1,sc4", expect=True,
+                                runas=ROOT_USER)
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+        # delete sc3 sched
+        self.server.manager(MGR_CMD_DELETE, SCHED, id="sc3", expect=True,
+                                runas=ROOT_USER)
+        
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="sc3", expect=True,
+                                runas=ROOT_USER)
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="", expect=True,
+                                runas=ROOT_USER)
+        
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc2", expect=True,
+                                runas=ROOT_USER)
+
+
+        # delete sc1 sched
+        self.server.manager(MGR_CMD_DELETE, SCHED, id="sc1", expect=True,
+                                runas=ROOT_USER)
+
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="sc1", expect=True,
+                                runas=ROOT_USER)
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
```

Comment:
You've already done this test.  You deleted sched sc3 and tested you couldn't list it.  No need to do it again for sc1.